fixtures/webhook_call/find.yaml

```diff
@@ -0,0 +1,72 @@
+interactions:
+- request:
+    body: null
+    headers:
+      Accept: ['*/*']
+      Accept-Encoding: [identity]
+      Authorization: [Bearer foobar]
+      Connection: [keep-alive]
+      Content-Type: [application/vnd.contentful.management.v1+json]
+      User-Agent: [python-requests/2.18.1]
+      X-Contentful-User-Agent: [sdk contentful-management.py/1.3.1; os macOS/16.5.0;
+          platform python/3.5.2;]
+    method: GET
+    uri: https://api.contentful.com/spaces/orzkxlxlq59d/webhooks/16ypL3XjNK6oreLPPoVBxI/calls/4KYbXQ9cg8CE2aqouWqY2i
+  response:
+    body: {string: "{\n  \"sys\": {\n    \"id\": \"4KYbXQ9cg8CE2aqouWqY2i\",\n    \"type\":
```

Comment:
I wouldn't want to do that, it would take a whole day just to re-record everything and make sure I don't break any test. If I want the response body I can fetch it anyway by debugging the request inline in the test, it was just that for one test I wasn't understanding why a field was not showing up, turns out I was expecting something that wasn't supposed to be there. Then I just left gzip off because I thought it a nice addition.